google/cloud/opentelemetry/internal/monitored_resource.cc

```diff
@@ -174,7 +176,7 @@ MonitoredResourceProvider GenericNode() {
           {"location",
            {{sc::kCloudAvailabilityZone, sc::kCloudRegion}, "global"}},
           {"namespace", {{sc::kServiceNamespace}}},
-          {"node_id", {{sc::kHostId}}},
+          {"node_id", {{sc::kHostId, sc::kHostName}}},
```

Comment:
Good catch, thanks!

Can we have a unit test for this?

https://github.com/googleapis/google-cloud-cpp/blob/90d72c0e6d81d9e54cdc1faf2f5044832d59997c/google/cloud/opentelemetry/internal/monitored_resource_test.cc#L337

I would probably change `TestCase` to something like `LocationTestCase`, and then introduce a `NodeIdTestCase`, and loop over:

```cc
for (auto l : location_tests) {
  for (auto n : node_tests) {
    // verify l.expected_location and n.expected_node_id
  }
}
```

Reply:
Done